src/components/Dashboard/AddressListSm.vue

```diff
@@ -210,26 +210,21 @@
 </template>
 
 <script lang="ts">
-// Libraries
-import { Component, Mixins, Prop } from 'vue-property-decorator'
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
 import { Getter, State } from 'vuex-class'
-
-// Mixins
 import { CommonMixin, CountriesProvincesMixin } from '@/mixins'
-
-// Interfaces
 import { OfficeAddressIF } from '@/interfaces'
-
-// Components
 import FirmsAddressList from './FirmsAddressList.vue'
 
 @Component({
-  components: {
-    // sub-components
-    FirmsAddressList
-  }
+  components: { FirmsAddressList },
+  mixins: [
+    CommonMixin,
+    CountriesProvincesMixin
+  ]
```

Comment:
My code style recommendation:
- if it's a single item, keep it on 1 line
- if it's multiple items, put them on separate lines